integration/env/vars.sh

```diff
@@ -178,3 +178,4 @@ _onedpl_scrip_path=$(dirname -- "$(rreadlink "${vars_script_name:-}")")
 DPL_ROOT=$(dirname -- "${_onedpl_scrip_path}") ; export DPL_ROOT
 CPATH=$(prepend_path "${DPL_ROOT}/linux/include" "${CPATH:-}") ; export CPATH
 PKG_CONFIG_PATH=$(prepend_path "${DPL_ROOT}/lib/pkgconfig" "${PKG_CONFIG_PATH:-}") ; export PKG_CONFIG_PATH
+CMAKE_PREFIX_PATH=$(prepend_path "${DPL_ROOT}/lib/cmake/dpl" "${CMAKE_PREFIX_PATH:-}") ; export CMAKE_PREFIX_PATH
```

Comment:
```suggestion
CMAKE_PREFIX_PATH=$(prepend_path "${DPL_ROOT}/lib/cmake/oneDPL" "${CMAKE_PREFIX_PATH:-}") ; export CMAKE_PREFIX_PATH
```

I tested it with /opt/intel/oneapi/dpl/2022.1.0/

Reply:
Yes, you are correct. Fixed.
Thanks.